ci/e2e-test.sh

```diff
@@ -1,20 +1,31 @@
 #!/usr/bin/env bash
 
-DOCKER_COMPOSE_PROJECT=${1:-akvolumen}
-LUMEN_URL=${2:-http://t1.lumen.local:3030/}
-LUMEN_USER=${3:-jerome}
-LUMEN_PASSWORD=${4:-password}
-CYPRESS_RECORD_KEY=${CYPRESS_RECORD_KEY:-}
+set -o errexit
+set -o nounset
 
-if [ "${DOCKER_COMPOSE_PROJECT}" == "akvolumen" ]; then
-    DOCKER_COMPOSE_ARGS=""
+DOCKER_COMPOSE_PROJECT="${1:-akvolumen}"
+LUMEN_URL="${2:-http://t1.lumen.local:3030/}"
+LUMEN_USER="${3:-jerome}"
+LUMEN_PASSWORD="${4:-password}"
+CYPRESS_RECORD_KEY="${CYPRESS_RECORD_KEY:-}"
+
+if [[ "${DOCKER_COMPOSE_PROJECT}" == "akvolumen" ]]; then
```

Comment:
I don't get why the duplication is better.

Reply:
It's not better, i was trying to fix the e2e tests (lots of changes, related and unrelated) and this seems to be executing something with _wrong_ arguments. I can revert it back to the original version

Reply:
Not too fuss about it. Just wanted to know if there was something wrong with the original approach, so that I did not made the same mistake again.

I understand that the original approach was ok and I am also ok with the current approach